ocs_ci/helpers/helpers.py

```diff
@@ -4738,3 +4738,75 @@ def flatten_multilevel_dict(d):
         else:
             leaves_list.append(value)
     return leaves_list
+
+
+# def verify_cephblockpool_status(
```

Comment:
Please remove commented out blocks of code.

Reply:
Addressed in latest commit.